encoder/libav_encoder.hpp

```diff
@@ -49,7 +50,7 @@ class LibAvEncoder : public Encoder
 
 	void videoThread();
 	void audioThread();
-
+	void nextSegment(int64_t timestamp_us, int64_t &update_variable);
```

Comment:
Do we know why this is showing up green? Just curious!!

Reply:
Its a new addition -  I created the function